gpt_researcher/llm_provider/image/image_generator.py

```
@@ -0,0 +1,440 @@
+    def _ensure_client(self):
+        """Ensure the Google GenAI client is initialized."""
+        if self._client is None:
+            try:
+                from google import genai
+                self._client = genai.Client(api_key=self.api_key)
+                logger.info(f"Initialized image generation with model: {self.model_name}")
+            except ImportError:
+                raise ImportError(
+                    "google-genai package is required for image generation. "
+                    "Install with: pip install google-genai"
+                )
```

Ruff (0.14.14)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)